crates/pathfinder/src/storage/state.rs

```diff
@@ -1323,14 +1339,15 @@ mod tests {
 
     mod starknet_blocks {
         use super::*;
+        use crate::storage::test_utils;
 
-        fn create_blocks() -> [StarknetBlock; 3] {
-            crate::storage::test_utils::create_blocks::<3>()
+        fn create_blocks() -> [StarknetBlock; test_utils::NUM_BLOCKS] {
```

Comment:
I was thinking there was a change coming to the create_blocks, which is why I made one test not really care as long as 2 blocks are always created. Is this is a necessary change though? I'll try to go through commit by commit.

Reply:
This PR makes the test data setup more complicated, with different types of transactions being created. I guess this is why @CHr15F0x made these changes here, since these functions take many more inter-dependent parameters.

Reply:
This comes from the revert as well.